docs/data/material/customization/theme-components/theme-components.md

```diff
@@ -77,17 +90,62 @@ const finalTheme = createTheme({
 
 {{"demo": "GlobalThemeOverrideCallback.js"}}
 
-### Using `sx` (experimental) syntax
+### Using the `sx` experimental syntax
```

Comment:
Is the `sx` syntax only "experimental" in this specific context? I think we should elaborate on this if it's going to be mentioned in the header.

Reply:
I believe it is! Experimental in the sense that we're using it inside the theme object.